notebooks/ceo_mapping_analysis.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>what if the file follows some weird structure (e.g. where s is 01 and 02 instead of 1 and 2)? I think it's probably simpler here to just let the user write down csv_path1 and csv_path2 or something like that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>